### ISSUE TITLE
Extra newlines are added after switch case statements on strings in C#

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1257]
Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60008-UNI-17253.cs" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60008-UNI-17253.cs"
index 793c0a8..ed93e70 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60008-UNI-17253.cs"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60008-UNI-17253.cs"
@@ -1,9 +1,11 @@
-// It adds an extra newline after each case 
+// It adds an extra newline after each case
 switch (someString)
 {
     case "a":
+
         return foo;
 
     case "b":
+
         return bar;
 }
```

Debug support files:
---
[60008-UNI-17253.cs](https://gist.githubusercontent.com/mihaipopescu/c6f4f2c4c3a360f9c3dfd4afad58e883/raw/793c0a8047508bf18f23e8efdc92b45806cf2f3d/60008-UNI-17253.cs)
[60008-UNI-17253.cs.log](https://gist.githubusercontent.com/mihaipopescu/c6f4f2c4c3a360f9c3dfd4afad58e883/raw/aada0ed1e302de962510f1ca564f61e161912112/60008-UNI-17253.cs.log)
[60008-UNI-17253.cs.unc](https://gist.githubusercontent.com/mihaipopescu/c6f4f2c4c3a360f9c3dfd4afad58e883/raw/4f7126a5ca2e99c13e612bf9cc5273b7eefa71db/60008-UNI-17253.cs.unc)
[Uncrustify.JamCS.cfg](https://gist.githubusercontent.com/mihaipopescu/c6f4f2c4c3a360f9c3dfd4afad58e883/raw/3493e9dee5daad683fcac4da5e00059570e536f0/Uncrustify.JamCS.cfg)
[UNI-17253.cs](https://gist.githubusercontent.com/mihaipopescu/c6f4f2c4c3a360f9c3dfd4afad58e883/raw/793c0a8047508bf18f23e8efdc92b45806cf2f3d/UNI-17253.cs)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by CDanU** [comment]
Can't reproduce this one.

**Comment by mihaipopescu** [comment]
I don't know, that doesn't fix the issue.

We have another note there where we've activated `nl_before_return=true`

> there's a bug in uncrustify where braces removed by mod_full_brace_if are still seen by this option, requiring a double-run of uncrustify to resolve :/

This can be considered an instability.

**Comment by gmaurel** [comment]
Fixed at db6adb6c69b238746c60233a423eac3df02c4073, Dec 20 20:25:54 2017
